Dockerfile-slim.template

```diff
@@ -1,6 +1,7 @@
 FROM buildpack-deps:jessie-curl
 
-RUN groupadd -r node && useradd -r -g node node
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --create-home node
```

Comment:
Because simplicity also has value.

Also, why is it a goal to have exactly 1000 as UID/GID and nothing else? Does the bug you referenced require that (it's too long and I didn't read all messages).